rnd/autogpt_server/autogpt_server/blocks/search.py

```diff
@@ -139,7 +139,7 @@ class Output(BlockSchema):
 
     def __init__(self):
         super().__init__(
-            id="f7a8b2c3-6d4e-5f8b-9e7f-6d4e5f8b9e7f",
+            id=str(uuid.uuid4()),  # 使用uuid库自动生成唯一ID
```

Comment:
We don't want to do this to allow users to share templates down the road. You can see that a bit in #7511